Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -843,6 +844,10 @@ internal async Task UpdateFormsInnerNavigation(Page pageBeingRemoved)
 		{
 			if (NavPage == null)
 				return;
+			if (_ignorePopCall) {
+				_ignorePopCall = false;
```

Comment:
What's the reason for setting the false here? opposed to up inside the OnPopViewAsync call after *actuallyRemoved = (poppedViewController == null) ? true : !await task;*

Is there a use case doing it here that I'm not thinking of?

Reply:
you're right that location is way better. I actually didn't understood what the task did because on first glance it looks like it just subscribes some event handlers. But It waits until the Pop is completed so it shouldn't be a fine location to reset _ignorePopCall.

Reply:
I updated the Pull Request and made sure the popped event works with the UITest.